src/components/Copy-to-clipbord.tsx

```diff
@@ -20,7 +20,7 @@ const CopyToClipboard = ({
 
   return (
     <div>
-      <button onClick={handleCopyClick}>
+      <button onClick={() => handleCopyClick()}>
```

Comment:
This is not necessary as handleCopyClick has no parameters. Actually Copy component has to be modified because it does not copy the text for now

Reply:
I have modified the copy component as well it does ,copy the text now

Reply:
> I have modified the copy component as well it does ,copy the text

Let's @hkirat look at it, because at the time copy had its purpose to be like that and not copy the content, but with the introduction of roles, it should be fine to copy the content.

Reply:
Ok sir